src/components/NcDialog/NcDialog.vue

```diff
@@ -287,6 +292,11 @@ export default defineComponent({
 		 */
 		const hasNavigation = computed(() => slots?.navigation !== undefined)
 
+		/**
+		 * The unique id of the nav element
+		 */
+		const navigationId = ref(GenRandomId())
```

Comment:
Does this needs to be a reference, could it not be just a plain value?

Reply:
It could. It seems that making a property in Composition a `ref` is the same common practice as adding all the values to `data` in Options API, even if it is a constant.

(Though, in Composition it is easier (cleaner) to have constant properties).